### PR TITLE
Modernize Python 2 code to get ready for Python 3

### PR DESCRIPTION
**Pull request by cclauss**
Some minimal, "safe" changes to make the code syntax compatible with both Python 2 and Python 3.
* Run __futurize --stage1__ http://python-future.org/futurize_cheatsheet.html

Python 3 syntax compatibility does __not__ mean that the port is complete.  More work will be require to complete the port to Python 3.

* Not yet fixed [in wawCommons.py](https://github.com/IBM/watson-assistant-workbench/blob/master/scripts/wawCommons.py#L96): __python3 -c "'\U'"__  # --> SyntaxError: (unicode error)

flake8 testing of https://github.com/ibm/watson-assistant-workbench on Python 3.6.3

$ __flake8 . --count --select=E901,E999,F821,F822,F823 --show-source --statistics__
```
decode bytes in position 0-1: truncated \UXXXXXXXX escape
                    elif replacementPair[1] == '\U':
                                              ^
```

### CODE REVIEW

File: scripts/update_all.py [hunk 1]
```diff
@@ -37,17 +39,17 @@
             if os.path.isfile(strParamsItem):
                 paramsAll+= ' -c '+strParamsItem
             else:
-                print('ERROR: Configuration file %s not found.', strParamsItem)
+                print(('ERROR: Configuration file %s not found.', strParamsItem))
```
Comment (Tereza-Pytelova) [inline scripts/update_all.py]: Why double brackets?

File: scripts/IntentData.py [hunk 2]
```diff
@@ -12,16 +12,23 @@
...
+from .wawCommons import eprintf
 from collections import OrderedDict
 
 '''
 Created on Jan 15, 2018
 @author: alukes
 '''
 
+try:
+    unicode        # Python 2
```
Comment (Tereza-Pytelova) [inline scripts/IntentData.py]: There is no `unicode` import.

Comment (cclauss) [reply scripts/IntentData.py]: __unicode__ is used on line 152.  Which is ok in Python 2 where __unicode__ is a builtin (no import required) but is not ok in Python 3 where __unicode__ was removed.

File: scripts/update_all.py (hunk 1 above)
Comment (cclauss) [reply scripts/update_all.py]: Try it and see (in Python 2)...
```python
print “a”, “b”
print(“a”, “b”)
print((“a”, “b”))
```

The first and last print the same thing but the middle one does not.

**Comment by cclauss** [comment]
Travis output is https://travis-ci.com/IBM/watson-assistant-workbench/builds/84299089#L525

File: scripts/update_all.py (hunk 1 above)
Comment (Tereza-Pytelova) [reply scripts/update_all.py]: For me in the Python 2.7.14 the second and the last are the same 
```
>>> print("a", "b")
('a', 'b')
>>> print(("a", "b"))
('a', 'b')
```
But anyway, this row should be rather `print('ERROR: Configuration file %s not found.' % strParamsItem)`

Comment (cclauss) [reply scripts/update_all.py]: Do you have `from __future__ import print_function` at the top of the file?

**Comment by Tereza-Pytelova** [comment]
For some reasons which are not clear to me, Travis runs not this PR (#3), but the #38 one. @cclauss could you please try to create a new pull request (and better to the devel branch)?

**Comment by cclauss** [comment]
Closing in favor of #147

**Pull request closed without merging**